lib/active_model_serializers/test/schema.rb

```diff
@@ -68,6 +85,10 @@ def response_body
           load_json(response.body)
         end
 
+        def request_params
+          request.env['action_dispatch.request.request_parameters']
```

Comment:
this is defined in Rails.  We want it to be only GET params or the POST (raw_params) depending on the request.method, I think. I don't think there's a scenario where they should be merged (as request.params does, IIRC)